client/components/date-range/index.js

```diff
@@ -154,6 +158,7 @@ export class DateRange extends Component {
 		if ( this.props.trackExternalDateChanges ) {
 			newState.startDate = this.props.selectedStartDate;
 			newState.endDate = this.props.selectedEndDate;
+			newState.selectedShortcutId = this.props.selectedShortcutId;
```

Comment:
```suggestion
			newState.selectedShortcutId = selectedShortcutId;
```

Should we do this instead? or it doesn't matter.

Reply:
Is there the variable `selectedShortcutId` in the scope?